p2p/discovery/bucket.go

```diff
@@ -0,0 +1,53 @@
+/**
+*  @file
+*  @copyright defined in go-seele/LICENSE
+ */
+package discovery
+
+import (
+	"github.com/seeleteam/go-seele/log"
+)
+
+const (
+	bucketSize = 16 // Kademlia bucket size
```

Comment:
do we need to rename bucketSize to BucketSize?

I just suppose all constant vars should be uppercase.

Reply:
no, constant with uppercase will be exported, this is the rule in golang

Reply:
nice, got it.